client/src/api/iqdata/Queries.ts

```diff
@@ -200,3 +201,50 @@ export function useGetIQData(type: string, account: string, container: string, f
     setFFTsRequired,
   };
 }
+
+export function reshapeFFTs(currentFFTSize: number, currentData: Float32Array[], newFFTSize: number): Float32Array[] {
```

Comment:
There is a file called rfFunctions in utils that might be a better fit to where put this function ( that ile also need to be renamed for the new naming convention to rf-functions.ts 

Not a big deal and nothing that I dont think that would hole up the merge.

Reply:
Test is in this dir though so should be fine here